openlibrary/plugins/openlibrary/js/SearchBar.js

```diff
@@ -27,19 +27,18 @@ const RENDER_AUTOCOMPLETE_RESULT = {
         return `
             <li>
                 <a href="${work.key}">
-                    <img src="//covers.openlibrary.org/b/id/${work.cover_i}-S.jpg"/>
+                    <img src="//covers.openlibrary.org/b/id/${work.cover_i}-S.jpg?default=https://dev.openlibrary.org/static/images/icons/avatar_book-sm.png" alt=""/>
```

Comment:
We'll want the production server here :)

```suggestion
                    <img src="//covers.openlibrary.org/b/id/${work.cover_i}-S.jpg?default=https://openlibrary.org/static/images/icons/avatar_book-sm.png" alt=""/>
```